docs-requirements.txt

```diff
@@ -3,12 +3,12 @@ sphinx-rtd-theme==2.0.0rc4
 sphinx-autodoc-typehints==1.25.2
 
 # Need to install the api/sdk in the venv for autodoc. Modifying sys.path
-# doesn't work for pkg_resources.
--e "git+https://github.com/open-telemetry/opentelemetry-python.git#egg=opentelemetry-api&subdirectory=opentelemetry-api"
--e "git+https://github.com/open-telemetry/opentelemetry-python.git#egg=opentelemetry-semantic-conventions&subdirectory=opentelemetry-semantic-conventions"
--e "git+https://github.com/open-telemetry/opentelemetry-python.git#egg=opentelemetry-sdk&subdirectory=opentelemetry-sdk"
--e "git+https://github.com/open-telemetry/opentelemetry-python-contrib.git#egg=opentelemetry-util-http&subdirectory=util/opentelemetry-util-http"
-./opentelemetry-instrumentation
+# doesn't work for pkg_resources. Used by .readthedocs.yml
```

Comment:
Is the pkg_resources hunk still relevant?

Reply:
I'm not sure. I suppose we still need install core packages to run in readthedocs